linux-setup/readme.txt

```diff
@@ -0,0 +1,16 @@
+Lokinet needs certain capabilities to run to set up a virtual network interface and provide a DNS server. The preferred approach to using this is through the linux capabilities mechanism, which allows assigning limited capabilities without needing to run the entire process as root.
```

Comment:
Can you move this file to something docs, perhaps `docs/linux-static-readme.txt` rather than creating a linux-setup top-level dir.

Reply:
Just keeping inline with the Mac and Windows distribution folder system we have x-setup

Reply:
Unlike the macos and windows, this is just a bit of documentation which is *also* useful for people building from source on linux, while the contents of the setup-* have patches and scripts needed to massage the build.

Reply:
Ok will move later